### PR TITLE
fix torch is not defined in engineV2.py

### PR DESCRIPTION
**Pull request by ccsuzzh**
image

### CODE REVIEW

File: engineV2.py [hunk 1]
```diff
@@ -21,8 +21,9 @@
         APITestCINNVSDygraph,
         APITestPaddleOnly,
     )
-    import torch
-    import paddle
+
+import torch
+import paddle
```
Comment (cangtianhuang) [inline engineV2.py:26]: 请首先检查：是否同步至最新的仓库版本，是否正确安装python与torch

torch 与 paddle 的导入是在子进程初始化时进行的（init_workers），在主进程直接导入会导致子进程无法在 gpu 间隔离

\* 此处的 import 仅是作为类型提示用

Comment (ccsuzzh) [reply engineV2.py:26]: 这个PR修复确实很草率。我刚刚同步最新代码测试，仍然有同样的问题，我的环境之前就安装好python和torch了，是后面同步了新的代码才出现的这个问题，这是我的测试命令：`python engineV2.py --accuracy=True --api_config_file="tester/api_config/temp.txt" >> "tester/api_config/test_log/log.log" 2>&1`
image
